### DISCUSSION TITLE
How to add Content-Encoding header to pre-compressed files with webpack-dev-server?

### DISCUSSION THREAD

**Discussion by SetoKaiba** [webpack/webpack#16748] [Q&A]
I'm devloping a frontend with Angular and Unity WebGL builds. Unity WebGL builds comes with pre-compressed files like .gz or .br. So I have to add Content-Encoding to pre-compressed files. But I didn't find away to achieve it. Is there a way to support pre-compressed files like .gz or .br?

I checked the similar post, but it comes no solution. I know that I can use third party app. But I think it should support pre-compressed files.
https://github.com/webpack/webpack/discussions/15593

**Comment by alexander-akait** [comment]
@SetoKaiba Sorry for delay, what do you mean? Webserver should handle their (if you setup ir), for dev server you can add a middleware, but what is the reason?
